bigip/resource_bigip_ltm_monitor.go

```diff
@@ -148,10 +170,12 @@ func resourceBigipLtmMonitorRead(d *schema.ResourceData, meta interface{}) error
 			d.Set("manual_resume", m.ManualResume)
 			d.Set("parent", m.ParentMonitor)
 			d.Set("name", name)
+			d.Set("username", m.Username)
+			d.Set("password", d.Get("password").(string))
```

Comment:
I'll add my comment to this PR as well:

I'm not really sure how to go about doing this better. The LTM will hash the password when it receives any string (so sending in a password hash does not help), and when you get the monitor config back from the LTM it will always be different from what is set in your .tf file.